src/main/java/com/iota/iri/network/TransactionCacheDigester.java

```diff
@@ -0,0 +1,20 @@
+package com.iota.iri.network;
+
+import net.openhft.hashing.LongHashFunction;
+
+
+public class TransactionCacheDigester {
+
+    private static final LongHashFunction TX_CACHE_DIGEST_HASH_FUNC = LongHashFunction.xx();
+
+    /**
+     * Computes the digest of the given transaction data.
+     *
+     * @param txBytes The raw byte encoded transaction data
+     * @return The the digest of the transaction data
+     */
+    public static long getTxCacheDigest(byte[] txBytes) {
+        return TX_CACHE_DIGEST_HASH_FUNC.hashBytes(txBytes);
+    }
+
+}
```

Comment:
Good SoC :-)

Reply:
I think class name and/or method name is not perfect yet as both are redundant.

Reply:
Yeah...
Java tends to get verbose :-(

Bothers me less, but feel free to improve if you like

Reply:
changed method name